### DISCUSSION TITLE
Handler failed to bind

### DISCUSSION THREAD

**Discussion by BeepPlayz** [rapid7/metasploit-framework#16073] [Q&A] [answered]
[-] Handler failed to bind to 157.**.*.***:8080:-  -           
[*] Started reverse TCP handler on 0.0.0.0:8080

**Comment by bcoles** [comment, marked as answer]
Presumably the value you have set for `LHOST` is not a valid IP address for any of your network interfaces.

The options you're looking for are `ReverseListenerBindAddress` and`ReverseListenerBindPort`.

https://www.corelan.be/index.php/2014/01/04/metasploit-meterpreter-and-nat/

**Reply by BeepPlayz** [reply]
Bro my I gave Right LHOST and I am using termux so I can't use apache2 server

**Reply by bcoles** [reply]
> Bro my I gave Right LHOST and I am using termux

Ensure nothing is already listening on `LHOST:8080`.

> so I can't use apache2 server

You don't need an apache2 server to run a handler.

**Reply by BeepPlayz** [reply]
Ok thank you
